src/web/util/DeviceInfo.js

```diff
@@ -33,9 +33,13 @@ module.exports = Class(function () {
 
   this.getChromeSize = function (isRotated) {
     var opts = this._opts;
-    var size = opts && opts.background
-             ? new Size(opts.background.width, opts.background.height)
-             : new Size();
+    var bg = opts && opts.background;
+
+    if (Array.isArray(bg)) {
+      bg = bg[0];
+    }
+
+    var size = bg ? new Size(bg.width, bg.height) : new Size();
```

Comment:
This works for now, but only under the assumption that rotated images will be all be the same size, but with the axes flipped. In the future, we should probably check against the rotated image dimensions?